Math/IntegerToRoman.swift

```diff
@@ -30,4 +30,18 @@
         
         return res
     }
-}
\ No newline at end of file
+}
+
+// Alternate solution
+class Solution {
```

Comment:
Could you please change the class name to something else? Solution is too generic